src/diffusers/loaders.py

```diff
@@ -1523,6 +1532,9 @@ def load_lora_into_text_encoder(
                 tries to not use more than 1x model size in CPU memory (including peak memory) while loading the model.
                 Only supported for PyTorch >= 1.9.0. If you are using an older version of PyTorch, setting this
                 argument to `True` will raise an error.
+            adapter_name (`str`, *optional*):
+                Adapter name to be used for referencing the loaded adapter model. If not specified, it will use
+                `default_{i}` where i is the total number of adapters being loaded
```

Comment:
If this is useful for particular situations I would not mind adding a line about that too. But not too strongly opinionated.

Reply:
Users would be passing `adapter_name` mostly, so the default case is not useful for any specific situations and all goos on this front.

Reply:
addressed in the latest commit